src/v2/mod.rs

```diff
@@ -2171,6 +2268,31 @@ impl Client {
         })
     }
 
+    /// Get the winning bakers of an historical `Epoch`.
+    /// The stream ends when there are no more rounds for the epoch specified.
+    /// This only works for epochs in at least protocol version 6.
+    pub async fn get_winning_bakers_epoch(
+        &mut self,
+        ei: impl Into<EpochIdentifier>,
```

Comment:
I just remembered that here it never makes sense to use Best or LastFinal, so it's a bit annoying that that's the type that is being used.

Can you at least add in the documentation? I know it says "historical" now, but that's very unclear. 

Also could you please elaborate on "this only works", like you did for other endpoints.

Reply:
Sure